src/neuroglancer/image_user_layer.ts

```diff
@@ -73,9 +74,10 @@ export class ImageUserLayer extends Base {
   dataType = new WatchableValue<DataType|undefined>(undefined);
   sliceViewRenderScaleHistogram = new RenderScaleHistogram();
   sliceViewRenderScaleTarget = trackableRenderScaleTarget(1);
-  volumeRenderingRenderScaleHistogram = new RenderScaleHistogram();
-  // unused
-  volumeRenderingRenderScaleTarget = trackableRenderScaleTarget(1);
+  volumeRenderingRenderScaleHistogram =
+      new RenderScaleHistogram(VOLUME_RENDERING_RESOLUTION_LOG_SCALE_ORIGIN);
+  volumeRenderingRenderScaleTarget =
+      trackableRenderScaleTarget(2 ** VOLUME_RENDERING_STARTING_LOG_SCALE);
```

Comment:
it would probably be better to add the validation directly in a modified `trackableRenderScaleTarget` rather than just in the widget --- that way the parameter remains within the valid range even if set via the JSON state/Python.